src/MSBuild/app.amd64.config

```diff
@@ -48,8 +48,8 @@
         </dependentAssembly>
         <dependentAssembly>
           <assemblyIdentity name="Microsoft.Build.CPPTasks.Common" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" />
-          <bindingRedirect oldVersion="16.0.0.0-16.10.0.0" newVersion="16.10.0.0" />
-          <codeBase version="16.10.0.0" href="..\..\..\Microsoft\VC\v160\Microsoft.Build.CPPTasks.Common.dll" />
+          <bindingRedirect oldVersion="16.0.0.0-17.0.0.0" newVersion="17.0.0.0" />
```

Comment:
Can app.config files read properties from the build? Or is it possible to define some <CppVersion> variable that will update each app.config? (not a blocking question)

Reply:
You still need codebase, I believe
<codeBase version="17.0.0.0" href="..\..\..\Microsoft\VC\v160\Microsoft.Build.CPPTasks.Common.dll" />

Reply:
On the other hand, how about just removing this redirect? It won't be needed for 17.0

Reply:
Yes, please remove Microsoft.Build.CPPTasks.Common from msbuild.exe.config at all. We don't want a redirect between major versions (and I plan to investigate if we can get rid of the redirect between minor versions as well)

Reply:
Should we leave it but make it not do anything (that is, oldVersion="17.0.0.0-17.0.0.0") rather than removing it entirely? I'm thinking of when we start updating it, at which point it would be nicer to have the version already there and be able to just modify it.

Reply:
No, please remove. My hope is that we won't need to add it again

Reply:
if you are removing it, you need to remove the whole segment
```
<dependentAssembly>
          <assemblyIdentity name="Microsoft.Build.CPPTasks.Common" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" />
          <bindingRedirect oldVersion="16.0.0.0-17.0.0.0" newVersion="17.0.0.0" />
          <codeBase version="17.0.0.0" href="..\..\Microsoft\VC\v160\Microsoft.Build.CPPTasks.Common.dll" />
        </dependentAssembly>
```